src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/RazorLSPTextViewConnectionListener.cs

```diff
@@ -104,12 +86,14 @@ public void SubjectBuffersConnected(ITextView textView, ConnectionReason reason,
             throw new ArgumentNullException(nameof(textView));
         }
 
+        InitializeStartupServices();
```

Comment:
I'm guessing the answer is yes, but just to be sure, does this class get constructed when a project is open, as distinct from when a Razor document is opened? Because I would have thought with the latter, the background compiler would not be able to do its job early enough.

Reply:
Happy with your early reviews. Yeah, this is something I need to revisit. Thanks for your comment!

Reply:
> I'm guessing the answer is yes

Or, and the answer was no 😬